src/PowerShellEditorServices/Services/Workspace/LanguageServerSettings.cs

```diff
@@ -353,6 +357,26 @@ public class CodeFoldingSettings
         }
     }
 
+    /// <summary>
+    /// Pester settings
+    /// </summary>
+    public class PesterSettings
+    {
+        public PesterSettings()
+        {
+        }
+
+        public PesterSettings(PesterSettings settings)
+        {
+            Pester5CodeLens = settings.Pester5CodeLens;
+        }
+
+        /// <summary>
+        /// Whether integration features specific to Pester v5 are enabled
+        /// </summary>
+        public bool Pester5CodeLens { get; set; }
```

Comment:
Talked this over with @rjmholt ... can you rename it to something not specific to Pester5?

Here's what we were thinking (not ranked):
* `EnableItCodeLens`
* `EnableSingleTestCodeLens`
* `EnableItBlockCodeLens`
* `EnableItScopeCodeLens`

Reply:
then in the vscode-powershell side, in the package.json say it's a "Pester 5 and up" only setting

Reply:
The thing here is that this enables the lens on both It, and Context including child Contexts.

I would propose `EnableLegacyCodeLens = true`, with annotation in the vscode side that this enables code lens that is compatible with Pester 4. This will work but for now, when the legacy code lens is the default, and for the future, when Pester 5 will be the standard, and we switch this to `false`, and slowly deprecate it.

Reply:
@rjmholt and I are cool with your suggestion. 👍

Reply:
Done